scripts/config

```diff
@@ -87,6 +87,11 @@ set__conf() {
     fi
 }
 
+set__on_calendar() {
+    ynh_write_var_in_file --file="/etc/systemd/system/$app.timer"  --key=OnCalendar --value="$on_calendar"
+    systemctl daemon-reload
```

Comment:
Oh, probably you also need to update the setting in yunohost, otherwise during an upgrade you get this:
```
Warning: File /etc/systemd/system/borg.timer has been manually modified since the installation or last upgrade. So it has been duplicated in /var/cache/yunohost/appconfbackup//etc/systemd/system/borg.timer.backup.20250813.111833
Warning: --- /var/cache/yunohost/appconfbackup//etc/systemd/system/borg.timer.backup.20250813.111833	2025-08-13 11:17:40.177046719 +0000
Warning: +++ /etc/systemd/system/borg.timer	2025-08-13 11:18:33.577682588 +0000
Warning: @@ -2,7 +2,7 @@
Warning:  Description=Run backup borg regularly
Warning:  [Timer]
Warning: -OnCalendar=Monthly
Warning: +OnCalendar=Daily
Warning:  [Install]
Warning:  WantedBy=timers.target
```